General queries/Network footprint.txt

```diff
@@ -0,0 +1,42 @@
+// Query 1 shows you any network communication happened from endpoints to an specific Remote IP or Remote URL
+// Ensure to update RemoteIP and RemoteURL variable.
+// For questions @MiladMSFT on Twitter or milad.aslaner@microsoft.com by email
+NetworkCommunicationEvents
+| where EventTime > ago(30d)
+| where RemoteIP == "IP ADDRESS GOES HERE"
+or RemoteUrl contains "URL GOES HERE"
+| project EventTime, ComputerName, ActionType, RemoteIP, RemoteUrl, InitiatingProcessFileName, InitiatingProcessCommandLine
+
+// Query 2 shows you any network communication that happened from endpoints through a specific file to an Remote IP or Remote URL
+//Ensure to update RemoteIP, RemoteURL and InitatingProcessFileName
+// For questions @MiladMSFT on Twitter or milad.aslaner@microsoft.com by email
+NetworkCommunicationEvents
+| where EventTime > ago(30d)
+| where RemoteIP == " IP ADDRESS GOES HERE"
```

Comment:
Please add parenthesis around the or

Reply:
If I use parenthesis around the or I get a syntax error:

![image](https://user-images.githubusercontent.com/19193423/42717661-e103a9a8-8702-11e8-9c1b-4865332f751e.png)

![image](https://user-images.githubusercontent.com/19193423/42717654-d99bbaa2-8702-11e8-824b-438a1b3587e4.png)

Reply:
Once this is sorted I will send a new push request. Can u help me understand what I do wrong here @TomerAlpert ?

Reply:
Should do: (Cond1 or cond2) and cond3

When you do: cond1 or cond2 and cond3
It actually runs: cond1 or (cond2 and cond3)